docker-compose.link.yaml

```diff
@@ -4,7 +4,7 @@ version: '3'
 # connect this service to sibling services from other repositories
 
 services:
-  ffc-demo-web:
+  app:

```

Comment:
Renamed service to "app" since "ffc-demo-web" is prefixed onto all the Docker resources automatically.